pkg/controllers/clusterset/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -62,28 +63,31 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	returnResult := reconcile.Result{
+		RequeueAfter: time.Second * 5,
```

Comment:
why do we need to requeue?

Reply:
because we always need to sync the clusterdeloyment lablel with clusterpool if it's not claimed, if the clusterpool's label changed, we need to sync the change from requeue.

Reply:
why not watch clusterpool and reconcile all related clusterdeployment?

Reply:
done